### ISSUE TITLE
Grid: tree-like grid when sort second column reverse row value in first column

### ISSUE THREAD

**Issue by wondertalik** [vitmalina/w2ui#1573]
When grid load is everything ok. But if i want to order the values in second column the rows in first column  change  reverse order.
Example, http://jsfiddle.net/h30m9zbL/3/ - click Families, order in Mes change too

**Comment by mrg2001** [comment]
Yes, I see the problem. I have not found a solution yet, but just a note, if you sort by first column and then hold ctrl (cmd) key, and click by second column, sort is as desired. I found that the problem is because of these lines
```js
                // break tie for similar records,
                // required to have consistent ordering for tree paths
                var ret = compareCells(a.recid, b.recid, -1, 'asc');
```
However, they cannot be deleted as it would break consistency of subrecords
